packages/address/src/__tests__/MultisigAddress.spec.ts

```diff
@@ -0,0 +1,85 @@
+import Address from '..'
+import { MultisigAddress } from '../MultisigAddress'
```

Comment:
This can just be `import MultisigAddress from '../MultisigAddress'` with the default export change